web/package.json

```diff
@@ -26,10 +26,13 @@
         "bccm-video-player": "^1.2.4",
         "graphql": "^16.6.0",
         "graphql-tag": "^2.12.6",
+        "sortablejs": "^1.14.0",
```

Comment:
If you don't use a library, don't add them to the package.json (and include in the commit)

Same with vue-drag-drop and vue-draggable. They are packages which do the same job if I'm not mistaken. (and also unused)